application/worker.py

```diff
@@ -449,7 +457,17 @@ def run_task(t, args, aggregate_model=False):
                 t.repo_dir = gherkin_repo_dir
 
         try:
+            from datetime import datetime
+            start_time = datetime.now()
             task(d, *args)
+            end_time = datetime.now()
+            if t.__name__ != "general_info_task":
+                vt_id = super(task_class_name, t).__getattribute__(task,"validation_task_id")
```

Comment:
I don't understand this.

Following the comment above, we should be able to just to `t.validation_task` or `t.validation_task_id` in the current version.